PsfLauncher/StartProcessHelper.h

```diff
@@ -58,9 +27,11 @@ HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR curren
     };
 
     PROCESS_INFORMATION processInfo{};
-    LPPROC_THREAD_ATTRIBUTE_LIST attributeList;
-    MakeAttributeList(attributeList);
-    startupInfoEx.lpAttributeList = attributeList;
+
+    if (attributeList)
+    {

```

Comment:
I think this if statement should be removed - it's okay to set startupInfoEx.lpAttributeList to nullptr.  It's not okay to leave startupInfoEx.lpAttributeList as stack garbage.